bindings/nodejs/lib/types/wallet/create-delegation-transaction.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Wallet, DelegationId } from '../..';
+import { PreparedTransaction } from './prepared-transaction';
+import { Type } from 'class-transformer';
+import { TransactionWithMetadata } from './transaction';
+import { PreparedTransactionData } from '../client/prepared-transaction-data';
+
+/*
+ * Represents prepared data for issuing a transaction to create a delegation.
+ */
+export class PreparedCreateDelegationTransaction extends PreparedTransaction {
+    readonly _delegationId: DelegationId;
```

Comment:
does that work with serialisation ?

Reply:
Well this type doesn't get serialized

Reply:
Hmm ok, it's a bit confusing to me why we have both PreparedCreateDelegationTransaction and PreparedCreateDelegationTransactionData tbh

Reply:
one is a helper class which contains the wallet reference, so you can just call `send()` on it, and thus it cannot be deserialized